go-controller/pkg/ovn/policy.go

```diff
@@ -316,6 +366,116 @@ func (oc *Controller) createDefaultDenyPortGroup(policyType knet.PolicyType) {
 	}
 }
 
+// Creates a policy to allow multicast traffic within 'ns':
+// - a port group containing all logical ports associated with 'ns'
+// - one "from-lport" ACL allowing egress multicast traffic from the pods
+//   in 'ns'
+// - one "to-lport" ACL allowing ingress multicast traffic to pods in 'ns'.
+//   This matches only traffic originated by pods in 'ns' (based on the
+//   namespace address set).
+func (oc *Controller) createMulticastAllowPolicy(ns string) {
```

Comment:
@dceara see comment below about returning errors from this function

Reply:
Done.